backend/Recodo.API/Startup.cs

```diff
@@ -21,6 +22,9 @@ public void ConfigureServices(IServiceCollection services)
         {
 
             services.AddControllers();
+            services
+                .AddMvcCore()

```

Comment:
I think we don't need everything from that extension. You can chain `AddFluentValidation()` after `AddControllers()`
And configure it with registering validators from the Common. For that you can create a new folder `Validators` and create an empty class that will be updated when we have actual DTOs and validatos